packages/lesswrong/lib/types/collectionTypes.ts

```diff
@@ -214,7 +214,7 @@ type EditableFieldsIn<T extends DbObject> = NonAnyFieldsOfType<T,EditableFieldCo
 type DbInsertion<T extends DbObject> = ReplaceFieldsOfType<T, EditableFieldContents, EditableFieldInsertion>
 
 type SpotlightDocumentType = 'Post' | 'Sequence';
-
+type CollectionNameType = 'Messages' | 'Comments';
```

Comment:
I'd probably give this a more specific name, since it doesn't really explain why `Messages` and `Comments` are collection names but not any of the others